src/frontend/src/lib/utils/address.utils.ts

```diff
@@ -0,0 +1,13 @@
+import type { AddressData } from '$lib/stores/address.store';
+import type { CertifiedStoreData } from '$lib/stores/certified.store';
+import type { Address, OptionAddress } from '$lib/types/address';
+import type { TokenId } from '$lib/types/token';
+
+export const getNullableAddress = ({
```

Comment:
The purpose of the function is not really to get a nullable address but rather to map the address regardless if certified or not certified. Do you have a better naming idea? Maybe just `mapAdress` or `addressData`?

Reply:
to keep consistent with similar ones (for example the ones related to tokens), I would call it `mapAddress` as you said (or `mapAddressData`)

Reply:
Let's do that

Reply:
done